spec/integration/_dev/benchmarks/scenario.spec.yml

```diff
@@ -0,0 +1,24 @@
+spec:
+  type: object
+  additionalProperties: false
+  required:
+    - version
+    - data_stream
+    - description
+    - corpora
+  properties:
+    version:
+      description: The package version to benchmark.
```

Comment:
For this field, it can be added also a pattern to match the value.
For instance in `version` field in the main manifest, there is an example of pattern (and examples) that could be used:

https://github.com/elastic/package-spec/blob/main/spec/integration/manifest.spec.yml#L212

Would it make sense to use `$ref` to point to the same definition as in that manifest.spec.yml ?